src/CalculatorUITests/StandardModeFunctionalTests.cs

```diff
@@ -304,5 +314,893 @@ public void Memory_ClearTest()
             Assert.IsNotNull(page.MemoryPanel.MemoryPaneEmptyLabel);
         }
         #endregion
+
+        #region Error Tests
+        [TestMethod]
+        public void Error_DivideByZero()
+        {
+            page.StandardOperators.InvertButton.Click();
+            Assert.IsTrue(page.GetCalculatorResultText() == "Cannot divide by zero");
+        }
+        [TestMethod]
+        public void Error_CubedNegativeNumber()
+        {
+            page.StandardOperators.NumberPad.Input(9);
+            page.StandardOperators.NegateButton.Click();
+            page.StandardOperators.SquareRootButton.Click();
+            Assert.IsTrue(page.GetCalculatorResultText() == "Invalid input");
+        }
+        #endregion
+
+        #region KOT(Keep on top) Tests
+        [TestMethod]
+        [Priority(0)]
+        public void KOT_EnterExitKeepOnTop()
+        {
+            /// <summary>
+            /// Test Case 23456961: BVT: KOT: Verify entering and exiting KOT mode
```

Comment:
If this is just for aligning to VSTS manual tests, I would remove these tags. It is likely they will get out of sync and these will no longer be valid.

Same goes for the below tests.

Reply:
In https://github.com/microsoft/calculator/pull/788/commits/ded929f19a56f1eef397c5864d4ec6fde59e78ab